content/en/fvm/basics/introduction/index.md

```diff
@@ -12,6 +12,50 @@ aliases:
 
 {{< beta-warning >}}
 
-For the best intro to the FVM, check out this presentation from FIL Lisbon 2022:
+## What's it for
+
+The FVM enables developers to build many new use cases on top of and with the Filecoin network.
+
+### Data access control
+
+One advantage centralized storage solutions have over web3 services is that they can limit individual data access.
+
+The FVM enables Filecoin network participants to do the same. Threshold encryption networks, like Medusa, can individually or programmatically govern who can access a data collection without any contracts or validators having access to the data themselves.
```

Comment:
@aashidham, I reduced the _aggressiveness_ of the first sentence here. I also removed the last sentence that reinforces the benefits of FVM + data access control. What're your thoughts here?